config.js

```diff
@@ -60,6 +60,8 @@ config.swagger = {
   basePath: config.url
 }
 
+config.userTypes = [ 'attendee', ' company', 'organization' ]
```

Comment:
I would say this should be an object instead of an array. Then we'd do `config.userType.attendee` instead of `userTypes[0]` in which isn't clear the type of type of user.